src/frontend/src/handler/create_function.rs

```diff
@@ -50,7 +50,7 @@ pub async fn handle_create_function(
         Some(lang) => {
             let lang = lang.real_value().to_lowercase();
             match &*lang {
-                "python" | "java" | "wasm" | "rust" | "javascript" => lang,
+                "python" | "java" | "wasm" | "rust" | "javascript" | "deno" => lang,
```

Comment:
I don't think we should introduce `deno` as a new language. From the user's perspective, they are just writing javascript. Normally they shouldn't care which backend the function will run on. But since now we have 2 backends available, how about introducing a new option `RUNTIME [quickjs | deno]` for setting this?

Reply:
Sounds good